crates/ark/src/modules/positron/graphics.R

```diff
@@ -134,107 +90,178 @@ renderWithPlotDevice <- function(filepath, format, width, height, res, type) {
     # Replace bindings.
     env_bind_force(baseenv(), ".Devices", .Devices)
     env_bind_force(baseenv(), ".Device", newDevice)
+
+    # Also set ourselves as a known interactive device.
+    # Used by `dev.interactive()`, which is used in `stats:::plot.lm()`
+    # to determine if `devAskNewPage(TRUE)` should be set to prompt before
+    # each new plot is drawn.
+    grDevices::deviceIsInteractive(name)
```

Comment:
This is new, without this `plot.lm()` doesn't stop and prompt the user to hit Enter

We still don't handle this quite right because we don't show any plots until the next top level expression idle time, but we can iterate on this.